wasm/jit/jit_amd64.go

```diff
@@ -1683,7 +1811,7 @@ func (c *amd64Compiler) callFunctionFromRegisterIndex(reg int16) {
 
 func (c *amd64Compiler) releaseAllRegistersToStack() {
 	used := len(c.locationStack.usedRegisters)
-	for i := len(c.locationStack.stack) - 1; i >= 0 && used > 0; i-- {
+	for i := uint64(0); i < c.locationStack.sp && used > 0; i++ {
```

Comment:
We generally ok on  `i++`? I don't remember why but some time ago someone said in go we are supposed to never use that and always laboriously do `i = i+1`

Reply:
> some time ago someone said in go we are supposed to never use that and always laboriously do i = i+1

I don't think so, we abuse `i++` or `i--` everywhere 😄